src/components/AgenciesPage/components/components/components/AgenciesItemImage/AgenciesItemImage.tsx

```diff
@@ -8,8 +8,12 @@ interface AgenciesItemImageProps {
 
 export const AgenciesItemImage: React.FC<AgenciesItemImageProps> = ({
   image,
-}: AgenciesItemImageProps) => (
-  <div className="agencies-item-image-wrapper">
-    <CardMedia component="img" image={image} alt="astronaut"></CardMedia>
-  </div>
-);
+}: AgenciesItemImageProps) => {
+  const img = image ? image : process.env.PUBLIC_URL + '/logo.png';
```

Comment:
you can set `/logo.png` as default parameter in props

Reply:
Agree 
I added a check for the parent component passing the prop 
Because from API I get image_url = null
and then in AgenciesItemImage default prop doesn't work

![image](https://user-images.githubusercontent.com/77783277/168471669-1d91341e-99e8-4417-b8b6-1d952a23e3e5.png)